emoji/core.py

```diff
@@ -121,7 +128,7 @@ def emojize(
     pattern = re.compile('(%s[%s]+%s)' %
                          (re.escape(delimiters[0]), _EMOJI_NAME_PATTERN, re.escape(delimiters[1])))
 
-    def replace(match):
+    def replace(match: Any) -> str:
```

Comment:
We need to use Any here, because re.Match is only available in python 3.8

Reply:
There is `Match[str]` from `typing`. It looks like it is deprecated in favor of `re.Match`, but at the moment in Python 3.13 it seems to be still available.